src/query/config/src/outer_v0.rs

```diff
@@ -1314,6 +1314,9 @@ pub struct QueryConfig {
 
     #[clap(skip)]
     quota: Option<TenantQuota>,
+
+    #[clap(long)]
+    pub allow_insecure_tenant: bool,
```

Comment:
How about `internal_enable_sandbox_tenant`? `internal_` to indicate that it's an internal only config.